testsuites/syncgateway/functional/tests/test_db_online_offline_webhooks.py

```diff
@@ -173,9 +173,6 @@ def test_db_online_offline_webhooks_offline_two(params_from_base_test_setup, sg_
 
     cluster.servers[0].delete_bucket("data-bucket")
 
-    log_info("Sleeping for 120 seconds...")
-    time.sleep(120)
-
```

Comment:
as it a single test, I do not see jenkins job run for this(sgw job u pasted as only sinlge test run which is not this test)  . As long test passed for you locally, then it is fine. There is neither retries nor config added to avoid sleep time. Can you run couple of times locally and see if it passed consistently? The reason is that this sleep might added to avoid inconsistent failures